modules/test/playwright/tests/commerce/commerceSpecificationFacet.spec.ts

```diff
@@ -0,0 +1,138 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {commercePagesTest} from '../../fixtures/commercePagesTest';
+
+export const test = mergeTests(apiHelpersTest, commercePagesTest);
+
+test('can sort specifications by specification group priority', async ({
+	apiHelpers,
+	commerceLayoutsPage,
+	specificationFacetsPage,
+}) => {
+	const pageLabel = 'Specification Facet Page';
+
+	await commerceLayoutsPage.goToPages();
+	await commerceLayoutsPage.createWidgetPage(pageLabel);
+	await specificationFacetsPage.goToPage();
+	await specificationFacetsPage.addRequiredFacetWidgets();
+	await specificationFacetsPage.configureSearchOptions();
```

Comment:
try to move this block after all of the API calls, maybe this way you can avoid the reload.

Reply:
Hi @andrea-ale-sbarra ,

The reload is necessary. The page has to reload, otherwise the locators will not be found.